README.md

```diff
@@ -95,20 +110,19 @@ All edits and updates to existing Open Liberty Docs must be documented and track
 
    Link the PR to the issue. Request a review of the PR from David Mueller (`dmuelle`), Charlotte Holt (`Charlotte-Holt`) or one of [the doc maintainers](https://github.com/orgs/OpenLiberty/teams/docs-maintainers/members). Work with your reviewer to update the staging nav.adoc file.
 
-   In the PR, provide a link to your update on the [draft site](https://draft-openlibertyio.mybluemix.net/docs/).
+   In the PR, provide a link to your update on the [docs-draft site](https://docs-draft-openlibertyio.mybluemix.net/docs/) or [full draft site](https://draft-openlibertyio.mybluemix.net/docs/).
    
-8. If any changes are requested to the PR, make them in your branch and push them to draft first. Then, run the draft site build from Travis CI again to check that they look right on the draft site. Open a new PR to staging and request another review.
+8. If any changes are requested to the PR, make them in your personal branch and push them to `draft` first. After the PR is merged, wait for IBM Cloud to rebuild the [docs-draft site](https://docs-draft-openlibertyio.mybluemix.net/docs/) or [full draft site](https://draft-openlibertyio.mybluemix.net/docs/) and verify the changes.  If you have access to [this Slack channel](https://app.slack.com/client/T15GKHBT4/C01GDRYT4UC), you can use it to track build/deploy progress.
 
-9. After the PR to staging is approved and merged, request a build of the [staging openliberty.io site from Travis CI](https://travis-ci.com/github/OpenLiberty/openliberty.io). 
-    1. Click **More Options > Trigger Build**. Make sure the `staging` branch is selected, then click **Trigger custom build**. The staging site build starts running.
+9. After the PR to `staging` is approved and merged, IBM Cloud will automatically rebuild the [docs-staging site](https://docs-staging-openlibertyio.mybluemix.net/docs/) and [full staging site](https://staging-openlibertyio.mybluemix.net/docs/). If you have access, you can track the progress in the [Slack channel](https://app.slack.com/client/T15GKHBT4/C01GDRYT4UC).  
 
-10. When the build finishes, check to make sure the doc renders correctly on the [staging site](https://staging-openlibertyio.mybluemix.net/docs/). If any changes are needed make sure to add them to the draft branch and review on the draft site before you make a new PR to staging.
+10. When the build finishes, check to make sure the doc renders correctly on the [docs-staging site](https://docs-staging-openlibertyio.mybluemix.net/docs/) or [full staging site](https://staging-openlibertyio.mybluemix.net/docs/). If any changes are needed make sure to add them to the `draft` branch and review on the [full draft site](https://draft-openlibertyio.mybluemix.net/docs/) or [docs-draft site](https://docs-draft-openlibertyio.mybluemix.net/docs/) before you make a new PR to `staging`.
```

Comment:
If any changes are needed make sure to --->
If any changes are needed, make sure to